scripts/scil_label_and_distance_maps.py

```diff
@@ -51,11 +51,11 @@ def main():
     assert_outputs_exist(parser, args, [args.output_label,
                                         args.output_distance])
 
-    is_header_compatible(args.in_bundle, args.in_centroid)
-
     sft_bundle = load_tractogram_with_reference(parser, args,
                                                 args.in_bundle)
 
+    is_header_compatible(sft_bundle, args.in_centroid)
```

Comment:
here args.in_centroid can also be a tck right ?

Reply:
Indeed ! Fixed in 32dfe8f